routes/web.php

```diff
@@ -51,3 +52,20 @@
     $users = DB::table('login_users')->where('id', $id)->get();
     return $users;
 });
+
+Route::post('/user/{id}', function($id){
+    $users = DB::table('login_users')->where('id', $id)->get();
```

Comment:
there should be something called count() or size() or something. direct db function, we should use that